.github/workflows/build-test.yml

```diff
@@ -94,7 +94,12 @@ jobs:
         export HELM_NAMESPACE=ess
         for checkov_values in charts/matrix-stack/ci/*checkov*values.yaml; do
           echo "Testing matrix-stack with $checkov_values";
-          checkov -d "charts/matrix-stack" --framework helm --var-file "$checkov_values" --quiet
+          mkdir checkov-tmp
+          cd checkov-tmp
+          helm template checkov -f "../$checkov_values" ../charts/matrix-stack | sed 's/{{//' | yq -s '.kind + "-" + .metadata.name'
+          checkov -d .  --skip-check CKV_SECRET_6 --quiet
```

Comment:
We've got suppression annotations in various files, do we also need `--skip-check` here.

Reply:
`CKV_SECRET_6` applies to all resources (including ConfigMaps, for which we dont have annotations)

I'm fine in keeping skip-check annotations per-workload, but we can also skip the check globally. In which case we probably don't need checkov values files anymore. What do you think?

Reply:
Let's comment here why we're skipping globally and then remove the individual skips from the values files.

However all these changes make it harder to test locally as we can't just run `checkov` anymore. I suspect we should extract a script that takes a values file path and does all this

Reply:
Sorry I think there's been some confusion, I was only meaning remove the `checkov.io/skip4: CKV_SECRET_6=Postgres annotations hash are triggering CKV_SECRET_6.` skips as everything else can be done on a targetted per-workload basis.